File: Backend/src/main/java/com/ssafy/pickit/domain/vote/dto/VoteRequest.java
package com.ssafy.pickit.domain.vote.dto;

public record VoteRequest(
	Long memberId,
	String voteSessionId,
	String candidateId,
	String transactionHash
) {
	public VoteRequest withMemberId(Long newMemberId) {
		return new VoteRequest(newMemberId, this.voteSessionId, this.candidateId, this.transactionHash);
	}
}

File: Android/app/src/main/java/com/ssafy/pickit/data/datasource/remote/request/vote/VoteRequest.kt
package com.ssafy.pickit.data.datasource.remote.request.vote

import com.google.gson.annotations.SerializedName

data class VoteRequest(
    @SerializedName("vote_session_id")
    val voteSessionId: String,
    @SerializedName("candidate_id")
    val candidateId: Long,
    @SerializedName("transaction_hash")
    val transactionHash: String
)
